src/olympia/users/models.py

```diff
@@ -189,6 +189,15 @@ def __init__(self, *args, **kw):
     def __str__(self):
         return u'%s: %s' % (self.id, self.display_name or self.username)
 
+    @classmethod
+    def get_lookup_field(cls, identifier):
```

Comment:
Would be nice to have separate test coverage for this method.

Reply:
Added in 909b5096b6.